src/cdi_analytic/Analytic_EICoupling.cc

```diff
@@ -26,7 +26,7 @@ namespace rtt_cdi_analytic {
  *        rtt_cdi_analytic::Analytic_EICoupling_Model object
  */
 Analytic_EICoupling::Analytic_EICoupling(SP_Analytic_Model model_in)
-    : analytic_model(model_in) {
+    : analytic_model(std::move(model_in)) {
```

Comment:
the move is safe in this case?

Reply:
is it safe because you're copying in anyway, so you might as well move in the copy?

Reply:
That's right! Sorry to chime in, @keadyk and I were looking at this yesterday in another clang-tidy change. It's an optimization in certain cases to allow the compiler to use the move constructor. Explained better here:
https://clang.llvm.org/extra/clang-tidy/checks/modernize-pass-by-value.html

Reply:
Thanks! As I thought more about, it made more sense, but definitely a new pattern for me to learn. The article was super helpful, gracias!